GoMeet/src/main/webapp/js/error-messages.js

```diff
@@ -6,3 +6,7 @@ const INVALID_TIME_ERROR = 'The dates and times entered should be unique and be
 const INSUFFICIENT_REQUEST_PARAM = 'Please provide all required parameters to the GET request';
 const INVALID_PARAM_TYPE = 'Invalid parameter type';
 const INVALID_PARAM_VALUE = 'Invalid parameter value. Please check that the parameters are of the correct format and range';
+const ENTITY_NOT_FOUND = 'This entity could not be found';
+const SAME_TITLE = 'There is already a location with this title';
+const MAX_ENTITIES = 'The maximum number of entities has been reached';
```

Comment:
"... Buy GoMeet Premium to get 10 Locations per Meeting." 😬

Reply:
nit - 'The maximum number (5) of entities has been reached'

Reply:
There we go, we have our revenue model 😂

Reply:
@vivianho18 Hmm if the '5' is included in the message, the error message would need to be manually updated with every change in the MAX_ENTITIES value. 
[Optional] Maybe somehow it can be refactored to `MAX_ENTITIES = 'The maximum number of entities ' + MAX_NUMBER + ' has been reached'.` ?

Reply:
(note: I think this is a good suggestion and will try to implement it, however, I think it has low priority so I will come back to it)